jujugui/static/gui/src/app/app.js

```diff
@@ -1110,6 +1110,7 @@ YUI.add('juju-gui', function(Y) {
         var charm = app.db.charms.getById(service.get('charm'));
         var relatableApplications = utils.getRelatableApplications(
           this.db, models.getEndpoints(service, this.endpointsController));
+        const ecs = this.env.get('ecs');
```

Comment:
That's great, but are you sure the ecs object is a constant? Oh, I read now that "the const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned". Great, this will confuse me forever...